content/en/guide/v10/upgrade-guide.md

```diff
@@ -220,7 +219,7 @@ We renamed/moved the following properties:
 - `nodeName` -> `type`
 - `children` -> `props.children`
 
-As much as we tried, we always ran into edge-cases with third-party libraries written for react. This change to our `vnode` shape removed many difficult to spot bugs and makes our `compat` code a lot cleaner.
+As much as we tried, we always ran into edge-cases with third-party libraries written for react. This change to our `vnode` shape removed many difficulties to spot bugs and makes our `compat` code a lot cleaner.
```

Comment:
The previous translation was correct. There is a minor difference:

- "many difficult to spot bugs" -> many bugs that are hard to find
- "many difficulties in spotting bugs" -> running into many problems while looking for bugs

Reply:
@marvinhagemeister 
Thank you for review.
I undo this change.